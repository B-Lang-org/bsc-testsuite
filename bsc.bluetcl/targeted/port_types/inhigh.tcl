namespace import ::Bluetcl::*

puts [flags set {-verilog}]
puts [module load sysInhighEnable]

puts {----------}

puts [submodule ports sysInhighEnable]
puts {}
puts [submodule porttypes sysInhighEnable]

puts {----------}

puts [module ports mkInhighEnable_Sub]
puts {}
puts [module porttypes mkInhighEnable_Sub]

puts {----------}

