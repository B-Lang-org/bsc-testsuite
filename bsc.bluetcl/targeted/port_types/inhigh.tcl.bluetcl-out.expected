-verilog
mkInhighEnable_Sub sysInhighEnable
----------
{rg1 MOD {interface {{method _write _write {clock _clk__1} {reset _rst__1} {args {{{port Q_IN} {size 1}}}}} {method _read _read {clock _clk__1} {reset _rst__1} {args {}} {result D_OUT}}}} {args {{reset _rst__1 {port RST} {clock _clk__1}} {clock _clk__1 {osc CLK} {gate CLK_GATE}}}}} {rg2 mkInhighEnable_Sub {interface {{method _write _write {clock default_clock} {reset no_reset} {args {{{port _write_1} {size 1}}}}} {method _read _read {clock default_clock} {reset no_reset} {args {}} {result _read}}}} {args {{clock default_clock {osc CLK}} {reset default_reset {port RST_N} {clock default_clock}}}}}

{rg1 MOD {ports {{RST Reset} {CLK Clock} {CLK_GATE Bool} {Q_IN Bool} {D_OUT Bool}}}} {rg2 mkInhighEnable_Sub {ports {{CLK Clock} {RST_N Reset} {_write_1 Bool} {_read Bool}}}}
----------
{interface {{method _write _write {clock default_clock} {reset no_reset} {args {{{name _write_1} {port _write_1} {size 1}}}}} {method _read _read {clock default_clock} {reset no_reset} {args {}} {result _read}}}} {args {{clock default_clock {osc CLK}} {reset default_reset {port RST_N} {clock default_clock}}}}

{CLK Clock} {RST_N Reset} {_write_1 Bool} {_read Bool}
----------
